clients/ops/admin-ui/src/features/common/DaysLeftTag.tsx

```diff
@@ -0,0 +1,37 @@
+import { Tag } from "@fidesui/react";
+import React from "react";
+
+type DaysLeftTagProps = {
+  days_left: number | undefined;
```

Comment:
Given that this is an optional attribute, you can do the following:

`days_left?: number`

I would rename the attribute to `daysLeft` to be camel case.